index.html

```diff
@@ -325,7 +331,40 @@ <h3 id="legendtext">I'm always interested in hearing about new projects, so if y
       <li><a href="#"><i class="fab fa-medium-m box five foot"></i></a></li>
     </ul>
   </main>
+  <!-- <button class="openPopUp">Click Me!</button> -->
+  <!-- <div class="modal-parent">
+      <div class="modal">
+          <div class="modal-head">
+              <div class="modal-title">Multi Post Stories</div>
+              <div class="close-icon">&times;</div>
+          </div>
+          <div class="modal-body">
+            <ul class="modal-list">
+              <li class="tag csstag">css</li>
+              <li class="tag htag">html</li>
+              <li class="tag bootag">bootstrap</li>
+            </ul>
+            <div class="modal-image"><img class="modal-img" src="images/Snapshoot Portfolio.png" alt=""></div>
+              <p class="modal-description">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Fugiat aut rem, doloremque ullam voluptates aliquam. Error provident rerum alias atque!</p>
+              <div class="buttons">
+                <button type="button" class="modal-btn see-live">
+                  <p class="btn-text">
+                    See live <i class="fas fa-mars"></i>
+                </p>
+                </button>
+                <button type="button" class="modal-btn see-github">
+                  <p>
+                    See source <i class="fab fa-github"></i>
+                  </p>
+                </button>
+                
+              </div>
+          </div>
+      </div>
+  </div> -->
```

Comment:
- [ ] Kindly remove the comments from your code, leaving unnecessary comments in your project make it messy and unreadable for the user and future reviewer